src/concrete/ml/torch/compile.py

```diff
@@ -181,16 +184,32 @@ def _compile_torch_or_onnx_model(
             for each input. By default all arguments will be encrypted.
         reduce_sum_copy (bool): if the inputs of QuantizedReduceSum should be copied to avoid
             bit-width propagation
+        composition_mapping (Optional[Dict]): Dictionary that maps output positions with input
```

Comment:
adding this new parameter to the private funct `_compile_torch_or_onnx_model` instead of the other public ones to keep things internal